exercises/practice/perfect-numbers/.meta/example.cpp

```diff
@@ -0,0 +1,40 @@
+#include <cmath>
+#include <numeric>
```

Comment:
What's that for?

Reply:
oops. I don't need it anymore.

Is there a tool that will warn of unnecessary `#include`s?

Reply:
You could use on of these: 
https://stackoverflow.com/questions/614794/detecting-superfluous-includes-in-c-c

Reply:
Thanks!